person.rb

```diff
@@ -0,0 +1,31 @@
+class Person
+  attr_accessor :id, :name, :age
```

Comment:
**Optional**
- Please notic;e;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;

Reply:
hi @MahmoudBakr23 ,

Thank you so much for taking the time to review my code on GitHub. Your feedback and suggestions have been incredibly valuable to me, and I appreciate your insights and expertise in this area.